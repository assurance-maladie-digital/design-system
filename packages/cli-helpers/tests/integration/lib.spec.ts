import * as cliHelpers from '@/index';

describe('lib', () => {
	it('should register all helpers', () => {
		expect(cliHelpers).toMatchSnapshot();
	});
});
