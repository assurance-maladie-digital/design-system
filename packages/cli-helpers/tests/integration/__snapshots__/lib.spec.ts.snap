// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib should register all helpers 1`] = `
Object {
  "cmd": [Function],
  "colors": Object {
    "lightGrey": "#909090",
    "primary": "#01579b",
  },
  "default": [Function],
  "done": [Function],
  "error": [Function],
  "event": [Function],
  "getPath": [Function],
  "info": [Function],
  "log": [Function],
  "renderHeader": [Function],
  "traceLine": [Function],
  "verbose": [Function],
  "warn": [Function],
}
`;
